plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -291,6 +291,7 @@ public List<JdbcColumnHandle> getColumns(ConnectorSession session, JdbcTableHand
             return tableHandle.getColumns().get();
```

Comment:
Should we handle this for other jdbc classes like `JdbcSplitManager`?

Reply:
All the other classes use `JdbcClient` it might either result in redundant checks